codeception/codeception/CVE-2021-23420.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Deserialization of Untrusted Data
+link:      https://github.com/advisories/GHSA-4574-qv3w-fcmg
+cve:       CVE-2021-23420
+branches:
+    "4.1":
+        time:     2021-08-06 17:20:00
+        versions: ['>=4.1.0', '<4.1.22']
+    "4.0":
+        time:     2021-08-06 17:20:00
```

Comment:
this should be `~` (i.e. `null`) as it is not patched in 4.0

Reply:
Alright, updated